autotx/wallets/safe_smart_wallet.py

```diff
@@ -17,4 +17,4 @@ def on_transactions_prepared(self, txs: list[models.Transaction]) -> None:
         pass
 
     def on_transactions_ready(self, txs: list[models.Transaction]) -> bool | str:
-        return self.manager.send_tx_batch(txs, not self.auto_submit_tx)
+        return self.manager.send_multisend_tx_batch(txs, not self.auto_submit_tx)
```

Comment:
do we want to send it as multisend even when there's just one transaction in the list?

Reply:
I changed it to not be multi send in case of a single tx